pootle/apps/pootle_store/models.py

```diff
@@ -223,18 +223,25 @@ def get_for_user(self, user):
         """Filters units for a specific user.
 
         - Admins always get all non-obsolete units
-        - Regular users only get units from enabled projects.
+        - Regular users only get units from enabled projects accessible
+        by the user.
```

Comment:
Nit: _accessible to them._. Mind the indent too. There's also an identical comment in _pootle_translationproject.models_ and a similar one in _pootle_project.models_